.travis.yml

```diff
@@ -20,7 +20,7 @@ env:
     - SESSION_RESET_THRESHOLD_TEST=2000
     - SUBSEQUENT_ALERT_MESSAGE_THRESHOLD_TEST=1000
     - RADAR_WINDOW_SIZE_SECONDS_TEST=2
-    - LOW_BATTERY_ALERT_TIMEOUT_TEST=2
```

Comment:
Ah! I wonder if this is why the tests are failing in Travis, @susanchung-brave . I think you accidentally deleted this environment variable.